src/components/Feedback/Feedback.scss

```diff
@@ -87,3 +87,11 @@ button.pf-v6-c-button.chr-c-button-feedback {
     border-bottom: 1px solid var(--pf-t--global--border--color--default);
   }
 }
+
+@media (max-height: 450px), (max-width: 700px) {
```

Comment:
@tahmidefaz if possible we should use patternfly's baked in break points for this? https://www.patternfly.org/developer-resources/global-css-variables/#breakpoint-variables-and-class-suffixes

<img width="883" height="549" alt="image" src="https://github.com/user-attachments/assets/39a2f0e1-33ba-44e8-863e-d5387dbc0289" />

Reply:
@florkbr Thanks! Didn't know that was a things. Updated the media query condition with breakpoint vars. 👍